KakaoImageSearch/Data/Remote/SearchRemoteDataSource.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  SearchRemoteDataSource.swift
+//  KakaoImageSearch
+//
+//  Created by 강현석 on 2021/03/09.
+//
+
+import Alamofire
+import RxSwift
+
+struct SearchRemoteDataSource : SearchRemoteDataSourceType {
```

Comment:
굳이 제안하자면 콜론 사이 띄어쓰기..
```suggestion
struct SearchRemoteDataSource: SearchRemoteDataSourceType {
```